analytics/mapreduce/src/main/java/mil/nga/giat/geowave/analytic/mapreduce/kde/KDEJobRunner.java

```diff
@@ -431,4 +444,19 @@ protected int getCQLFilterArg() {
 		return 15;
 	}
 
+	protected void addJobClasspathDependencies(
+			final Job job,
+			final Configuration conf )
+			throws IOException,
+			URISyntaxException {
+		final String[] jars = conf.getTrimmedStrings(GEOWAVE_CLASSPATH_JARS);
```

Comment:
I'm able to pass it in from the invoking application.  For example:

    final Configuration conf = new Configuration();
    final String[] hdfsPaths = new String[] {...};
    conf.setStrings(KDEJobRunner.GEOWAVE_CLASSPATH_JARS, hdfsPaths);
    
    final String[] args = new String[] {...};
    ToolRunner.run(
        conf,
        new KDEJobRunner(),
        args);

So my application will upload the JARs to HDFS, and just pass the paths in through the configuration.